vars/mtaBuild.groovy

```diff
@@ -38,12 +63,19 @@ def call(Map parameters = [:]) {
     }
 }
 
-private getMtaJar(parameters) {
+private getMtaJar(parameters, customStepConfiguration, defaultStepConfiguration, configuration) {
+
     def mtaJarLocation = 'mta.jar' //default, maybe it is in current working directory
 
-    if(parameters?.mtaJarLocation){
-        mtaJarLocation = "${parameters.mtaJarLocation}/mta.jar"
-        echo "[mtaBuild] MTA JAR \"${mtaJarLocation}\" retrieved from parameters."
+    if(configuration?.mtaJarLocation){
```

Comment:
I understand the wish to figure out from where the jarLocation came from. But in the other hand it is weird to merge a configuration and later on hand over the different levels of configuration into the retrieval method just for logging.
That is the one thing: The other thing is: imaging the mtaJarLocation is provided from different levels of configuration. Of course the closest wins. But in the log we would have retrieval statements for all the configuration levels. This would be false and confusing.

Reply:
Yes, I do not like it too, but how can I do it? So far, I just follow the format that there was before. I could remove the method. Of course if the configuration levels change this if/else statement would be false ..

Reply:
IMO is would be sufficent to handover the configuration map which as been merged before, of course and to emit something like "[mtaBuild] MTA JAR \"${mtaJarLocation}\" retrieved from configuration.".